jbehave-support-core/src/main/java/org/jbehavesupport/core/ws/WebServiceHandler.java

```diff
@@ -53,10 +55,12 @@
  * }
  * </pre>
  */
+
 public abstract class WebServiceHandler {
 
     private static final String CONTEXT_SEPARATOR = ".";
     private static final String REQUEST_POSTFIX = "Request";
+    public static final String BRACKET_REGEX = "(.*)\\[(\\d+)\\](\\.)(.*)";
```

Comment:
Nitpicking: do we need the selection groups if we dont use them? Might improve readability a bit

Reply:
From the code perspective they are unnecessary, however I used them to make sense of the regex, because it is a cluster of escaping special characters